recipe/meta.yaml

```diff
@@ -19,6 +23,7 @@ requirements:
     - setuptools
   run:
     - python
+    - certifi
 
 test:
```

Comment:
Added a small test on the recipe to check that certifi is picked up by libcloud.